packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -243,38 +309,52 @@ const Select = <
     [options],
   );
 
-  const renderValue = useCallback(
-    (selected: Value) => {
-      // If the selected value isn't an array, then we don't need to display
-      // chips and should fall back to the default render behavior
-      if (typeof selected === "string") {
-        return undefined;
-      }
-
-      // Convert the selected options array into <Chip>s
-      const renderedChips = selected
-        .map((item: string) => {
-          const selectedOption = normalizedOptions.find(
-            (option) => option.value === item,
-          );
-
-          if (!selectedOption) {
-            return null;
+  const Chips = ({
+    selection,
+    isInteractive,
+  }: {
+    selection: string[];
+    isInteractive: boolean;
+  }) => (
+    <ChipsInnerContainer
+      isInteractive={isInteractive}
+      odysseyDesignTokens={odysseyDesignTokens}
+    >
+      {selection.map((item: string) => (
+        <MuiChip
+          key={item}
+          label={
+            <>
+              {item}
+              {!isInteractive &&
+                controlledStateRef.current === CONTROLLED &&
+                hasMultipleChoices && (
+                  <NonInteractiveIcon
+                    odysseyDesignTokens={odysseyDesignTokens}
+                  />
+                )}
```

Comment:
Does this need to be memoized?

Reply:
I don't think so, since `isInteractive` won't change for the lifespan of the component